internal/provider/deployment.go

```diff
@@ -114,7 +114,7 @@ func (d *DeploymentDeployer) assignNodesIPs() error {
 		return errors.Wrapf(err, "invalid ip %s", d.IPRange)
```

Comment:
there's a difference between IP and IPRange, let's be more specific in the wrapped error